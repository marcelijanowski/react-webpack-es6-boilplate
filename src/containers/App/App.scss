@import '../../assets/styles/colour_semantics';

.app-header {
  color: $header;
}