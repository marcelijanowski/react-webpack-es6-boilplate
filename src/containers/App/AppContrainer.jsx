import React, { Component } from 'react';
import './App.scss';

class App extends Component {
  render() {
    return (
      <div className="app">
        <h1 class="app-header">Hello World</h1>
      </div>
    );
  }
}

export default App;
