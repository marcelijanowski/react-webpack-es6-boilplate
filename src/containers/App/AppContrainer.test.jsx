import React, { Component } from 'react'
import renderer from 'react-test-renderer';

import App from './AppContrainer';

describe('App container', () => {
  it('renders a snapshot', () => {
    const tree = renderer.create(<App/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
})