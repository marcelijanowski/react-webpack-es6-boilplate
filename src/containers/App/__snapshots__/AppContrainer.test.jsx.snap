// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App container renders a snapshot 1`] = `
<div
  className="app"
>
  <h1
    className="app-header"
  >
    Hello World
  </h1>
</div>
`;
